diff --git a/vst3/.gitignore b/vst3/.gitignore
new file mode 100644
index 0000000..567609b
--- /dev/null
+++ b/vst3/.gitignore
@@ -0,0 +1 @@
+build/
diff --git a/vst3/buildLinux.sh b/vst3/buildLinux.sh
index 647969a..02f781c 100755
--- a/vst3/buildLinux.sh
+++ b/vst3/buildLinux.sh
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-VST_SDK_DIR=../../VST_SDK
+VST_SDK_DIR="${VST_SDK_DIR:-../../VST_SDK/vst3sdk}"
 
 rm -rf build
 
 mkdir -p build/asid.vst3/Contents
 cp -R asid.vst3/Contents/Resources build/asid.vst3/Contents
 
-mkdir -p build/asid.vst3/Contents/x86_64-linux
+mkdir -p build/asid.vst3/Contents/$(uname -m)-linux
 
 g++ \
 	src/vst3/entry.cpp \
@@ -47,11 +47,10 @@ g++ \
 	-Isrc \
 	-I$VST_SDK_DIR/vst3sdk/ \
 	-fPIC -shared \
-	-lxcb \
-	-o build/asid.vst3/Contents/x86_64-linux/asid.so \
-	-O3 \
-	-ffast-math
-	
+	$(pkg-config --cflags --libs xcb) \
+	-o build/asid.vst3/Contents/$(uname -m)-linux/asid.so \
+	${CXXFLAGS:--O3 -ffast-math} $LDFLAGS
+
 # for a more static build we probably need
 # -static-libgcc -static-libstdc++ -l:library.a...
 # with static libraries built with -fPIC
