diff --git a/RackSDK.cmake b/RackSDK.cmake
index 1e5a103..e3781d1 100644
--- a/RackSDK.cmake
+++ b/RackSDK.cmake
@@ -10,12 +10,6 @@ else ()
   message(STATUS "Using Rack-SDK in '${RACK_SDK_DIR}'")
 endif ()
 
-if (EXISTS "${RACK_SDK_DIR}/include/rack.hpp")
-  message(STATUS "Found include/rack.hpp")
-else()
-  message(FATAL_ERROR "Rack SDK in ${RACK_SDK_DIR} missing include/rack.hpp")
-endif()
-
 if ("${PLUGIN_NAME}" STREQUAL "")
   message(FATAL_ERROR "PLUGIN_NAME variable not set! Add PLUGIN_NAME variable to the project CMakeLists.txt before including RackSDK.cmake.\
  The PLUGIN_NAME must correspond to the plugin slug, as defined in plugin.json.")
