From 99775d9448da103efde869193db51a56bb27ddbe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20H=C3=BClsmann?= <fh@cbix.de>
Date: Sun, 8 Jun 2025 11:12:46 +0200
Subject: [PATCH] Add missing cstdint include

---
 include/core/timeseq-script.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/core/timeseq-script.hpp b/include/core/timeseq-script.hpp
index 9a7e493..183f7ed 100644
--- a/include/core/timeseq-script.hpp
+++ b/include/core/timeseq-script.hpp
@@ -3,6 +3,7 @@
 #include <string>
 #include <memory>
 #include <vector>
+#include <cstdint>
 
 namespace timeseq {
 
