--- CMakeLists.txt.orig	2026-03-18 13:33:39.896119646 +0100
+++ CMakeLists.txt	2026-03-18 13:39:57.192742269 +0100
@@ -5,7 +5,13 @@
 
 # This tells cmake we have goodies in the /cmake folder
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake-includes")
-include(GitVersion)
+if (NOT DEFINED ZL_EQ_VERSION)
+    include(GitVersion)
+else ()
+    add_definitions(-DZLEQUALIZER_CURRENT_VERSION="${ZL_EQ_VERSION}")
+    add_definitions(-DZLEQUALIZER_CURRENT_HASH="${ZL_EQ_HASH}")
+    file(WRITE VERSION "${ZL_EQ_VERSION}")
+endif ()
 include(PamplejuceVersion)
 
 # Configures universal binaries and decides which version of macOS to support
