diff --git a/src/window/Window.cpp b/src/window/Window.cpp
index cea09497..4b22be60 100644
--- a/src/window/Window.cpp
+++ b/src/window/Window.cpp
@@ -674,22 +674,24 @@ int Window::getMods() {
 #if defined ARCH_LIN
 	// On Linux X11, get mods directly from X11 display, to support X11 key remapping
 	Display* display = glfwGetX11Display();
-	XkbStateRec state;
-	XkbGetState(display, XkbUseCoreKbd, &state);
-
-	// Derived from GLFW's translateState() from x11_window.c
-	if (state.mods & ShiftMask)
-		mods |= GLFW_MOD_SHIFT;
-	if (state.mods & ControlMask)
-		mods |= GLFW_MOD_CONTROL;
-	if (state.mods & Mod1Mask)
-		mods |= GLFW_MOD_ALT;
-	if (state.mods & Mod4Mask)
-		mods |= GLFW_MOD_SUPER;
-	if (state.mods & LockMask)
-		mods |= GLFW_MOD_CAPS_LOCK;
-	if (state.mods & Mod2Mask)
-		mods |= GLFW_MOD_NUM_LOCK;
+	if (display != NULL) {
+		XkbStateRec state;
+		XkbGetState(display, XkbUseCoreKbd, &state);
+
+		// Derived from GLFW's translateState() from x11_window.c
+		if (state.mods & ShiftMask)
+			mods |= GLFW_MOD_SHIFT;
+		if (state.mods & ControlMask)
+			mods |= GLFW_MOD_CONTROL;
+		if (state.mods & Mod1Mask)
+			mods |= GLFW_MOD_ALT;
+		if (state.mods & Mod4Mask)
+			mods |= GLFW_MOD_SUPER;
+		if (state.mods & LockMask)
+			mods |= GLFW_MOD_CAPS_LOCK;
+		if (state.mods & Mod2Mask)
+			mods |= GLFW_MOD_NUM_LOCK;
+	}
 #else
 	// Use GLFW key codes on other OS's
 	if (glfwGetKey(win, GLFW_KEY_LEFT_SHIFT) == GLFW_PRESS || glfwGetKey(win, GLFW_KEY_RIGHT_SHIFT) == GLFW_PRESS)
